.github/workflows/postman-metrics.yml

```diff
@@ -0,0 +1,51 @@
+name: Compare and Send Postman Fork Data
+
+on:
+  workflow_dispatch:
+  workflow_call:
+    secrets: # all secrets are passed explicitly in this workflow
+      SLACK_WEBHOOK_URL:
+        required: true
+
+jobs:
+  compare-and-send-forks:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Fetch previous week's data from cache
+      uses: actions/cache/restore@v4
+      with:
+        path: ./tools/postman/openapi/fork-data.json
+        key: fork-data-${{ github.run_id }}
+        restore-keys: |
+          fork-data-
+
+    - name: Save previous week's data to file
```

Comment:
why did we decide to use previous week instead of something more recent?

Reply:
I should have changed the name to previous runs data since it can also be triggered manually. Stephen wanted weekly updates, which is why the cron is set to weekly.